parm/config/config.base.emc.dyn

```diff
@@ -224,6 +225,8 @@ if [[ "$CDUMP" == "gdas" ]] ; then
    export FTSFS=10
 elif [[ "$CDUMP" == "gfs" ]] ; then
    export FHCYC=24
+elif [[ "$CDUMP" == "gefs" ]] ; then
+   export FHCYC=24
```

Comment:
May be replace lines 223-230 with `case ${CDUMP} in` statements and condense `gfs` and `gefs`

Reply:
OK. I will do it.